documentation/specs/BuildCheck/Codes.md

```diff
@@ -14,6 +14,7 @@ Report codes are chosen to conform to suggested guidelines. Those guidelines are
 | [BC0201](#bc0201---usage-of-undefined-property) | Warning | Project | 9.0.100 | Usage of undefined property. |
 | [BC0202](#bc0202---property-first-declared-after-it-was-used) | Warning | Project | 9.0.100 | Property first declared after it was used. |
 | [BC0203](#bc0203----property-declared-but-never-used) | None | Project | 9.0.100 | Property declared but never used. |
+| [BC0301](#bc0301---building-from-downloads-folder) | None | Project | 9.0.300 | Building from Downloads folder. |
```

Comment:
If we might expand this in the future, should we already call it "untrusted folder" instead? And leave the messages as Downloads folder for now.

Reply:
It's very easy to edit this doc - no need for pushing any updates to users (as opposed to namings in code) - so I'd prefere to keep it very up to date with current state.